src/tests/fixtures/trip.fixture.js

```diff
@@ -16,7 +16,7 @@ export const trip = {
 const newTrip = {
   originId: faker.random.number({ min: 1, max: 9 }),
   destinationId: faker.random.number({ min: 1, max: 9 }),
-  departureDate: faker.date.future(),
+  departureDate: '2020-02-03',
```

Comment:
Any reason for this change?

Reply:
I had 2 choices, either changing it from the test file or from the fixture one and that's what I chose

Reply:
And you couldn't use faker to generate the date in future?

Reply:
Let me use it instead